### ISSUE TITLE
No more saving Enfold forms??

### ISSUE THREAD

**Issue by BrianaCunningham** [mdsimpson/contact-form-7-to-database-extension#30]
I have used the older version of this plugin for ages across multiple websites and now with new installations I don't see any way to save contact form 7 submissions and Enfold forms (which it used to have). Is this feature completely gone or am I missing a set up step? I need to be able to save Enfold forms or I can't use this plugin anymore. :(

**Comment by niguli** [comment]
I also need that Enfold theme forms are saved from your plugin ! Does it stil work with Enfold forms ?

**Comment by mdsimpson** [comment]
I don't have a license to the latest Enfold theme, so I'm not actively maintaining that integration.

**Comment by niguli** [comment]
Hi, I could easily ask **Kriesi** team (Enfold developers & owner) if I can give you my licence  number in order to check the last integration. It will be very useful for me. Let me know if I time to check it.

**Comment by mdsimpson** [comment]
Yes I'll see if I can update it if you let me use the license for that purpose.

**Comment by niguli** [comment]
Kriesi gave the ok (http://www.kriesi.at/support/topic/funny-but-really-important-question/?avia_support_contact=avs_59c3c2334c9322.61994912_854918). Can you send me you email to strumia @ gmail . com? I  will give you my license code. :)

**Comment by mdsimpson** [comment]
I made updates for Enfold integration and pushed them in CFDB version 2.10.24. 
You should see the update available in your WP Plugins page (assuming you installed [GitHub updater](https://github.com/afragen/github-updater/releases))

**Comment by ikaiser** [comment]
Hi all guys, I developed this plugin to solve this Question -> https://wordpress.org/plugins/contact-form-db-for-enfold/
